test/quantization/pt2e/test_generate_numeric_debug_handle.py

```diff
@@ -80,3 +81,43 @@ def test_quantize_pt2e_preserve_handle(self):
         repeated_debug_handle_ids = [2, 3, 6]
         for dh_id in repeated_debug_handle_ids:
             self.assertEqual(res_counter[dh_id], 2)
+
+    def test_copy_preserve_handle(self):
+        m = TestHelperModules.Conv2dThenConv1d()
+        example_inputs = m.example_inputs()
+        m = capture_pre_autograd_graph(m, example_inputs)
```

Comment:
can we use the new _export_for_training API? the current API here is already unmaintained.

Reply:
changed the call to `torch.export.export`, except for the quantization one, that has some issues with `model.meta`